Mathlib/Analysis/Calculus/LineDeriv/Basic.lean

```diff
@@ -344,10 +344,8 @@ theorem Filter.EventuallyEq.lineDifferentiableWithinAt_iff_of_mem
   h.lineDifferentiableWithinAt_iff (h.eq_of_nhdsWithin hx)
 
 lemma HasLineDerivWithinAt.congr_of_eventuallyEq (hf : HasLineDerivWithinAt 𝕜 f f' s x v)
-    (h'f : f₁ =ᶠ[𝓝[s] x] f) (hx : f₁ x = f x) : HasLineDerivWithinAt 𝕜 f₁ f' s x v := by
-  apply HasDerivWithinAt.congr_of_eventuallyEq hf _ (by simp [hx])
-  have A : Continuous (fun (t : 𝕜) ↦ x + t • v) := by fun_prop
-  exact A.continuousWithinAt.preimage_mem_nhdsWithin'' h'f (by simp)
+    (h'f : f₁ =ᶠ[𝓝[s] x] f) (hx : f₁ x = f x) : HasLineDerivWithinAt 𝕜 f₁ f' s x v :=
+  (EventuallyEq.hasLineDerivWithinAt_iff (EventuallyEq.symm h'f.eventually) hx.symm).mp hf
```

Comment:
Can you use dot notation here, as in `h'f.eventually.symm` or even `h'f.eventually.symm.hasLineDerivWithinAt_iff`?

Reply:
What about this (untested)?
```suggestion
  h'f.symm.hasLineDerivWithinAt_iff hx.symm |>.mp hf
```
I think going from `EventuallyEq` to `Eventually` was the mistake in the above suggestion.